docs/ECLStandardLibraryReference/SLR-Mods/DecodeBase64.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="DecodeBase64">
+  <title>DecodeBase64</title>
+
+  <para><emphasis role="bold">STD.Str.DecodeBase64<indexterm>
+      <primary>STD.Str.DecodeBase64</primary>
+    </indexterm><indexterm>
+      <primary>Str.DecodeBase64</primary>
+    </indexterm><indexterm>
+      <primary>DecodeBase64</primary>
+    </indexterm>(</emphasis> <emphasis>value</emphasis> <emphasis
+  role="bold">)</emphasis> <emphasis role="bold"></emphasis></para>
+
+  <informaltable colsep="1" frame="all" rowsep="1">
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry><emphasis>value</emphasis></entry>
+
+          <entry>A STRING value containing the data to decode.</entry>
+        </row>
+
+        <row>
+          <entry>Return:<emphasis> </emphasis></entry>
+
+          <entry>DecodeBase64 returns a DATA value.</entry>
+        </row>
+      </tbody>
+    </tgroup>
+  </informaltable>
+
+  <para>The <emphasis role="bold">DecodeBase64 </emphasis>function returns a
```

Comment:
You just said above it returns a data value... 

and I think you mean 'returns the DATA value obtained by decoding the supplied base64 string' or similar